doc/api.md

```diff
@@ -1198,11 +1237,38 @@ _**Request headers**_
 
 _**Request payload**_
 
-A JSON object with a `services` field. The value is an array of config groups objects to create. See the
+A JSON object with a `configGroups` or `services` field. The value is an array of config groups objects to create. See the
 [config group datamodel](#service-group-datamodel) for more information.
 
 Example:
 
+
+```json
+{
+    "configGroups": [
+        {
+            "resource": "/deviceTest",
+            "apikey": "801230BJKL23Y9090DSFL123HJK09H324HV8732",
+            "type": "Light",
+            "trust": "8970A9078A803H3BL98PINEQRW8342HBAMS",
+            "cbHost": "http://orion:1026",
+            "commands": [{ "name": "wheel1", "type": "Wheel" }],
+            "attributes": [
+                {
+                    "name": "luminescence",
+                    "type": "Integer",
+                    "metadata": {
+                        "unitCode": { "type": "Text", "value": "CAL" }
+                    }
+                }
+            ],
+            "lazy": [{ "name": "status", "type": "Boolean" }]
+        }
+    ]
+}
+```
+OR
+
```

Comment:
This should replace the existing example, not adding a new one

Reply:
Fixed in https://github.com/telefonicaid/iotagent-node-lib/pull/1375/commits/11787050de050581b6b76221f20d9c2c671c3b11